ts/lib/BUILD.bazel

```diff
@@ -22,21 +16,38 @@ py_binary(
         requirement("stringcase"),
     ],
 )
+_i18n_generated = [
+    "translate.ts",
```

Comment:
No strong feelings here, but 'translate' feels a bit odd to me as it's a verb. Would you be open to a noun like 'strings' or 'messages' instead?